pyne/mesh.py

```diff
@@ -704,6 +704,17 @@ def __init__(self, mesh=None, structured=False,
             stored in self.dims.
 
         """
+        # if Mesh is made and no parameters passed, raise MeshError
+        if (mesh is None) and (not structured) and (structured_coords is None) \
+            and (structured_set is None) and (structured_ordering=='xyz') \
```

Comment:
[PEP8](https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements): We'll want some spaces around the `==` here.